test/data/validation/destinations/ga4.json

```diff
@@ -317,6 +317,10 @@
           {
             "provider": "ketch",
             "consents": []
+          },
+          {
+            "provider": "iubenda",
+            "consents": []
```

Comment:
Can you please add some valid consent values?

Reply:
Resolved